app/cdap/components/RecipeList/RecipeTableRow.tsx

```diff
@@ -165,13 +163,26 @@ export const RecipeTableRow = ({
 
   return (
     <>
-      <div className="grid-row" onClick={!showAllColumns ? selectRecipeHandler : undefined}>
+      <div
+        className="grid-row"
+        onClick={!showAllColumns && selectRecipeHandler}
+        data-testid={`${recipe.recipeName
```

Comment:
create a reusable function like https://github.com/cdapio/cdap-ui/pull/870/files#:~:text=export%20const%20getTestIdString%20%3D%20(label%3A%20string)%20%3D%3E
and import this wherever required for generating testid
across pr

Reply:
used reusable function for generating testid